### PR TITLE
[CSR-2066] feat: get full test suite from JSON file

### PR DESCRIPTION
**Pull request by miguelangaranocurrents**
## Description

> Provide a brief description of the changes in this PR. Important to list all the changes made in overall. Describe any improvements, follow up tasks or edge cases related to this PR

This PR removes the full test suite creation in the upload command for the JUnit scanner. It will only read a JSON file with the full test suite.

## PR Checklist

- [x] I performed manual tests or added tests that prove my fix is effective or that my feature works.
- [x] I have performed a self-review of my code.
- [x] I have annotated my PR with comments, particularly in hard-to-understand areas.
- [x] I have considered the security implications of this work.

## Release Plan

> Do we need to update any environment variables? Is there any order of releases required?

- Package release

## Demo

> Add screenshots or videos demonstrating the solution if applicable

https://www.loom.com/share/6e9720910f224ff68cb93863f09659bb?sid=97690a7b-93e5-4e30-a99b-dea791962c77

## Manual Testing

> Describe in steps (support it with images if needed) how to try the changes of this PR. (Even the start/setup of the services needed to try it out). If it's a bug also include reproduction steps

1. Use the convert command to create instance files
2. Paste in the results folder the full test suite related to those results
3. Use the upload command

### CODE REVIEW

File: packages/cmd/src/lib/fs.ts [hunk 1]
```diff
@@ -16,8 +16,16 @@ export async function readJsonFile<T>(filePath: string): Promise<T> {
 
 export async function writeFileAsync(filePath: string, content: string) {
   try {
-    await fs.writeFile(filePath, content, 'utf8');
-    return filePath;
```
Comment (vCaisim) [inline packages/cmd/src/lib/fs.ts:20]: Conflict with the changes in #139

File: packages/cmd/src/services/upload/discovery/junit/scanner.ts [hunk 2]
```diff
@@ -1,21 +1,7 @@
-import fs from 'fs-extra';
```
Comment (vCaisim) [inline packages/cmd/src/services/upload/discovery/junit/scanner.ts:1]: Conflict with the changes in #139

File: packages/cmd/src/services/upload/index.ts [hunk 3]
```diff
@@ -223,7 +223,7 @@ function getMarkerFilePath(reportDir: string) {
   return path.join(reportDir, 'upload.marker.json');
```
Comment (vCaisim) [inline packages/cmd/src/services/upload/index.ts:223]: Conflict with the changes in #139

**Review by vCaisim — CHANGES_REQUESTED** [review]
Great that you checked how the `upload` command works with the changes. I think that this PR should be closed because the changes are already in #139
